scripts/ci/run-jenkins.sh

```diff
@@ -109,6 +109,15 @@ then
 	wget -qO- https://download.mono-project.com/test/new-certs.tgz| tar zx -C ~/.config/.mono/
 fi
 
+if [[ ${CI_TAGS} == *'ccache'* ]];
+then
+	# CCACHE_DIR should be set to a directory outside the chroot which holds the cache
+	CCACHE=`which ccache`
+	if [ "$CCACHE" ]; then
+		export EXTRA_CONF_FLAGS="${EXTRA_CONF_FLAGS} CC='ccache gcc' CXX='ccache g++'"
```

Comment:
Export the `CC` and `CXX` variables instead, similar to https://github.com/mono/mono/pull/11305/files#diff-4847efbf06c7d2913d13c1a50c2ae0b8L22

Reply:
Done.